### DISCUSSION TITLE
Consider moving this project to github.org/ddev

### DISCUSSION THREAD

**Discussion by rfay** [php-perfect/ddev-intellij-plugin#231] [Ideas]
This project has been so successful and well-maintained - it might be easier to find and have even more credibility if it were in the "ddev" org. Nothing would change in regards to ownership or maintenance.

**Comment by nico-loeber** [comment]
My open source licenses are licensed for "PHP Perfect", I am not sure if I might loose them, when moving the repository to another organization.
Other things are the secrets... I guess ddev orga owner will have access to them.

Right now I don't see that much of an advantage in moving the repository as it will probably cause some work and I will loose a bit of control.
I don't think many people are looking for it on github, they find the plugin by the JetBrains store.

**Reply by rfay** [reply]
You bet! The bottom line is, THANK YOU for creating and maintaining this! Whenever you think it's time, it will be welcome.

**Comment by gitressa** [comment]
It looks like it has since moved to https://github.com/ddev/, so perhaps a comment can be left about that, and link to an issue where it was decided, and the discussion can be closed?

**Comment by stasadev** [comment]
Resolved in:

- #387

**Comment by gitressa** [comment]
Nice, thanks @stasadev!
